packages/framework/tinylicious-client/src/test/TinyliciousClient.spec.ts

```diff
@@ -277,4 +283,112 @@ describe("TinyliciousClient", () => {
         await corruptedAliasOp(runtimeOf(do1), "alias");
         assert(await dataCorruption);
     });
+
+    /**
+     * Scenario: Test if TinyliciousClient starts the container in write mode. TinyliciousClient will attempt
+     * to start the connection in write mode, but if the write permission is not available or the file is read
+     * only, mode will be read.
```

Comment:
In your description can you be specific for this use case what plays out. I would assume you only need two use cases: one for write and other for read permissions.

Reply:
I have removed it because this is redundant with the read and write permission test.